openstack/identity/v3/users/requests.go

```diff
@@ -204,6 +204,45 @@ func Update(client *gophercloud.ServiceClient, userID string, opts UpdateOptsBui
 	return
 }
 
+// ChpwdOptsBuilder allows extensions to add additional parameters to
+// the ChangePassword request.
+type ChpwdOptsBuilder interface {
+	ToUserChpwdMap() (map[string]interface{}, error)
+}
+
+// ChpwdOpts provides options for changing password for a user.
+type ChpwdOpts struct {
+	// OriginalPassword is the original password of the user.
+	OriginalPassword string `json:"original_password,omitempty"`
```

Comment:
It looks like the Keystone API will try to authenticate with the original password:

https://github.com/openstack/keystone/blob/a6adc72e3e7ff4ff0bff0702e69ebd8f697d6261/keystone/identity/core.py#L1387

So wouldn't that mean `OriginalPassword` is required? If so, this field should be:

```go
OriginalPassword string `json:"original_password" required:"true"`
```